### DISCUSSION TITLE
How to know if PSD layer is Visible or Hidden?

### DISCUSSION THREAD

**Discussion by SavageStyle** [ImageMagick/ImageMagick#1956] [General]
Get all layers:
`MagickImageCollection magickImages = new MagickImageCollection(fileName)`
Iterate through layers:
`foreach(var image in magickImages) { ... }`

How i can get information about layer visibility?

**Comment by dlemstra** [comment]
You should probably ask this in the Magick.NET repository. You should check if the `Compose` property of the `image` is `CompositeOperator.No`.
